exercises/minesweeper/minesweeper_test.py

```diff
@@ -10,114 +10,158 @@
 
 from minesweeper import board
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
+
```

Comment:
@smalley, can you move the version comment down by a line for consistency? (ie. two blank lines before, one after)

Reply:
Sure, no problem at all @N-Parsons, I can make that change and push an update.